File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/AcceptJoinSessionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

public class AcceptJoinSessionMessage {

	public static final String TYPE = "AcceptJoinSessionMessage";

	private String color;

	public AcceptJoinSessionMessage() {
	}

	public AcceptJoinSessionMessage(String color) {
		this.color = color;
	}

	public String getColor() {
		return color;
	}

	public void setColor(String color) {
		this.color = color;
	}
}

File: client/Client/Network/Messages/Session/AcceptJoinSessionMessage.cs
namespace LoadRunnerClient.Messages
{
    public class AcceptJoinSessionMessage
    {
        public const string TYPE = "AcceptJoinSessionMessage";

        public AcceptJoinSessionMessage()
        {
        }

        private string _color;

        public string color
        {
            get => _color;
            set => _color = value;
        }
    }
}